### PR TITLE
Update Documentation for `select` elements

### PR DESCRIPTION
**Pull request by swordsreversed**
### Reasons for making this change

Updating Documentation

If this is related to existing tickets, include links to them as well.

https://github.com/mozilla-services/react-jsonschema-form/issues/1254

### Checklist

* [x] **I'm updating documentation**
  - [x] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [ ] I've run `npm run cs-format` on my branch to conform my code to [prettier](https://github.com/prettier/prettier) coding style
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: playground/samples/simple.js [hunk 1]
```diff
@@ -22,6 +22,19 @@ module.exports = {
...
+      destination: {
+        type: "string",
+        title: "Where are you headed?",
+        enum: [
+          "GB",
+          "AU",
+          "France"
+        ],
+        enumNames: [
+          "United Kindom",
```
Comment (epicfaace) [inline playground/samples/simple.js]:
```suggestion
          "United Kingdom",
```

File: playground/samples/simple.js [hunk 2]
```diff
@@ -22,6 +22,19 @@ module.exports = {
...
+          "Australia"
```
Comment (epicfaace) [inline playground/samples/simple.js]: I believe enumNames should have 3 elements here to correspond with the enum values.

File: docs/form-customization.md [hunk 3]
```diff
@@ -76,6 +76,28 @@ render((
...
 
+#### Creating a `select` element
+
+To create a select element, you can use `enum`. The array elements will render as options. To add labels that differ from the option's values use `enumNames`:
+
+```js
+const schema = {
+  type: "string",
+  enum: ['AU', 'FR', 'GB'],
+  enumNames: ["Australia", "France", "United Kingdom"]
```
Comment (epicfaace) [inline docs/form-customization.md]: Could you change all instances of "United Kingdom" to "Great Britain", or change "GB" to "UK", just for consistency?

**Comment by glasserc** [comment]
I'm not crazy about this change. I think describing rjsf in terms of end-user-facing features is a mistake. rjsf is designed to let you render a form for a schema which describes your data, and data should not be defined in terms of user presentation.

**Pull request closed without merging**